### DISCUSSION TITLE
Link one package to many others, don't link others back or to eachother

### DISCUSSION THREAD

**Discussion by WillsterJohnson** [changesets/changesets#1169] [Q&A]
The docs don't seem to address this directly.

TL:DR; do the following configs do the same thing?
```json
{
  "linked": [
    ["package-a", "package-b"],
    ["package-a", "package-c"],
  ]
}
// vs
{
  "linked": [
    ["package-a", "(package-b|package-c)"],
  ]
}
```

The docs mention that micromatch globs are valid, however it doesn't address whether they are or are not permitted for use as the second argument.

The use case for this is one package which has several dependency packages it must be versioned with, however those dependencies must not be versioned every time the main package is versioned;

- version bump for `dep-1`: same bump for `main`, no bump for `dep-2`, `dep-3` (a dependency changed, therefore the package should reflect this)
- version bump for `main`: no bump for `dep-1`, `dep-2`, `dep-3` (the package changed, the dependencies don't care)
